src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -882,6 +882,15 @@ export default class FilingHistoryList extends Mixins(
     )
   }
 
+  /** Check whether to supprese notice of articles. */
+  private suppressNOA (displayName:string, prop: string): boolean {
+    // Only suppress NOA when it meets the following conditions
+    const type = this.getEntityType === CorpTypeCd.COOP
+    const name = displayName === 'Correction - Annual Report'
+    const isNOA = prop === 'noticeOfArticles'
```

Comment:
If this was the right solution then you should use existing getters and enums as much as possible.

Reply:
Yeah. I was looking for enums for the `displayName` and the document name, but we don't have them. They are however available in the backend.